packages/metadata/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This script builds the files in the main and module directories.
+
+rm main/networks/list.cjs
+rm module/networks/list.js
+
+touch main/networks/list.cjs
+touch module/networks/list.js
+
+echo '/* eslint-disable */  
```

Comment:
note: this is added here to prevent eslint from complaining

Reply:
use idiomatic way (aka. heredoc) instead:

```
cat > module/networks/list.js <<EOF
... the content ...
EOF
```